docker/local/README.md

```diff
@@ -0,0 +1,39 @@
+## Turbinia local stack using Docker
+Turbinia can be run locally without any Cloud components using Docker. It will use Redis, Celery and local disk to store data and perform message broker functionality.
+
+### Caveats
+rawdisk: As Turbinia uses the loop device to mount different types of evidence (eg raw disks) the host operating system should support the loop device. Linux is currently the only OS that supports the processing of raw disks. It is possible to process different evidence types, eg compresseddirectory, using the local stack on OSX or Windows.
```

Comment:
I'm not sure if we even want to mention OSX/windows here or not given the lack of testing (unless you've done more than I realize) -- Maybe we should move that reference to the bottom?

Reply:
I develop, run and test most of the code on OSX only. Only when running cloud jobs I use the terraform/gcp setup. I'll remove the reference as by removing it it's implicit.